### ISSUE TITLE
@svelte/enhanced-img should show A11y issues

### ISSUE THREAD

**Issue by MrGreenTea** [sveltejs/kit#11192]
### Describe the problem

When using `<enhanced:img` and for example forgetting to add the `alt` attribute no warning/error is shown using `svelte-check` or `eslint` with `svelte/valid-compile`.

I believe these a11y issues should be shown for `<enhanced:img` as well.

1. install  `@sveltejs/enhanced-img`
2. add `<enhanced:img src={...} />` somewhere
3. run `svelte-check`
4. get no warnings/error

### Describe the proposed solution

Have `<enhanced:img />` show the same a11y issues `svelte-check` / `eslint` would show for regular `<img />` tags

### Alternatives considered

manually grepping the output of `vite build` for warnings

### Importance

would make my life easier
